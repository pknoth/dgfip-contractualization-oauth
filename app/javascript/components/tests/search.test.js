import ReactTestRenderer from 'react-test-renderer'
import Search from '../search'

describe('components | Search', () => {
  describe('render', () => {
    const renderer = ReactTestRenderer.create(<Search />)
    it('should be defined', () => {
      expect(renderer).toBeDefined()
    })
    it('should match snapshot', () => {
      expect(renderer).toMatchSnapshot()
    })
  })
})
