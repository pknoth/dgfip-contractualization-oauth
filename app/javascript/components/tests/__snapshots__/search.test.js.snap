// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | Search render should match snapshot 1`] = `
<div
  className="form__group"
>
  <label
    htmlFor="search"
  >
    Tapez un mot-clef :
  </label>
  <div
    className="search__group"
  >
    <input
      className=" value="
      id="search"
      name="search"
      type="text"
    />
    <button
      aria-label="Recherche"
      className="overlay-button"
    >
      <svg
        className="icon icon-search"
      >
        <use
          xlinkHref="#icon-search"
        />
      </svg>
    </button>
  </div>
</div>
`;
